src/main/java/com/exadel/discount/dto/DiscountDTO.java

```diff
@@ -0,0 +1,31 @@
+package com.exadel.discount.dto;
+
+import com.exadel.discount.dto.validation.Create;
+import lombok.Data;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Null;
+import javax.validation.constraints.Size;
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+@Data
+public class DiscountDTO {
+    @Null(groups = Create.class)
+    private UUID id;
+    @NotNull(groups = Create.class)
+    private CategoryDTO category;
+    @NotBlank(groups = Create.class)
+    private String name;
```

Comment:
Add messages for validation errors please

Reply:
+